frontend/src/App.js

```diff
@@ -1,13 +1,21 @@
-import "./App.css";
-
-import InputSide from "./components/InputSide/InputSide";
-import OutputSide from "./components/OutputSide/OutputSide";
+import LeftPanel from "./components/LeftPanel/LeftPanel";
+import RightPanel from "./components/RightPanel/RightPanel";
 
 function App() {
```

Comment:
Let's use arrow functions to maintain consistency.

Reply:
Done